helpers/ExperimentDetails_helper.js

```diff
@@ -0,0 +1,64 @@
+const Helper = require('@codeceptjs/helper');
+
+class ExperimentDetails extends Helper {
+
+    // before/after hooks
+    /**
+     * @protected
+     */
+    _before() {
+        // remove if not used
+    }
+
+    /**
+     * @protected
+     */
+    _after() {
+        // remove if not used
+    }
+
+    async getCurrentActiveKeys() {
+        const {Puppeteer} = this.helpers;
+
+        return await Puppeteer.executeScript(async () => {
+            const keys = await evolv.client.getActiveKeys();
+            return keys.current;
+        })
+    }
+
+    updateContext(contextItems) {
```

Comment:
Not blocking, but updateContext and getContextItems don't really do anything anymore -- they call directly through to the evolv client and could be removed

Reply:
As per our chat, Puppeteer is required to access `evolv` on the page.